packages/react-devtools-scheduling-profiler/src/TabBar.js

```diff
@@ -0,0 +1,133 @@
+/**
```

Comment:
Let's use `react-devtools-shared/src/devtools/views/TabBar.js` and `react-devtools-shared/src/devtools/views/TabBar.css` rather than fork.